node/src/reactor/main_reactor/tests.rs

```diff
@@ -225,6 +240,21 @@ fn has_completed_era(era_id: EraId) -> impl Fn(&Nodes) -> bool {
     }
 }
 
+fn lowest_available_block_height_on_node(height: u64, node_id: NodeId) -> impl Fn(&Nodes) -> bool {
```

Comment:
This function's name is a bit confusing... but I have no ideas for a good one :grimacing: Something like `create_lowest_available_block_height_check`, as it creates a closure that checks something, maybe?

Reply:
Yeah, indeed it returns a predicate for checking the the specified height is the lowest available. Maybe if I rename it to `is_lowest_available_block_height_on_node` and also add a comment it would be in line with the other checks like `is_in_era` and `has_completed_era`. WDYT?

Reply:
Ah, I missed the other two similar functions.
Hm, so to bring this back in line, how about 'has_lowest_available_block_below_height`, to stress the property of the height we're looking for? Or perhaps `node_has_lowest_available_block_below_height`, to also stress that it only looks at a single node?

Reply:
Yup, sounds good to me, I'll make the change.

Reply:
renamed to `node_has_lowest_available_block_at_or_below_height` in https://github.com/casper-network/casper-node/pull/4149/commits/7c1334f55e7430f11baba52f630ea966be6b929a